modules/user_data/variables.tf

```diff
@@ -3,6 +3,11 @@
 variable "aws_region" {
   description = "AWS region where GraphDB is being deployed"
   type        = string
+
+  validation {
+    condition     = contains(["us-east-2", "us-east-1", "us-west-1", "us-west-2", "af-south-1", "ap-east-1", "ap-south-2", "ap-southeast-3", "ap-southeast-4", "ap-south-1", "ap-northeast-3", "ap-northeast-2", "ap-southeast-1", "ap-southeast-2", "ap-northeast-1", "ca-central-1", "eu-central-1", "eu-west-1", "eu-west-2", "eu-south-1", "eu-west-3", "eu-south-2", "eu-north-1", "eu-central-2", "il-central-1", "me-south-1", "me-central-1", "sa-east-1"], var.aws_region)
```

Comment:
There are even more regions than the list here. Better to remove this validation as it might get outdated when AWS adds/removes a region.

I think Terraform will break immediately if the region is missing/not supported.

Reply:
These are all the regions from AWS documentation, except Government regions. If you think they might add more, I will just remove this validation. Thank you.

Reply:
it might really be an overkill to have a validation for the region.

Reply:
I removed this validation. Thank you.